shell/initialize/app-extended.js

```diff
@@ -72,47 +71,18 @@ async function extendApp(vueApp) {
 
   await installInjectedPlugins(appPartials, vueApp);
 
-  // Wait for async component to be resolved first
```

Comment:
We do not use aysnc components in our application so all of this can be removed.